framework/platform/bpl/include/bpl_cfg.h

```diff
@@ -445,6 +468,14 @@ int bpl_cfg_get_beerocks_credentials(const int radio_dir, char ssid[BPL_SSID_LEN
 int bpl_cfg_set_beerocks_credentials(const int radio_dir, const char ssid[BPL_SSID_LEN],
                                      const char pass[BPL_PASS_LEN], const char sec[BPL_SEC_LEN]);
 
+/**
+ * Returns the platform SDL policy
+ *
+ * @return mem_only_psk on success
```

Comment:
what does `mem_only_psk` mean here and why is it the only return value? I think this could be explained better (by the engineer who implemented this obviously)